### DISCUSSION TITLE
nginx-proxy-manager disable ipv6?

### DISCUSSION THREAD

**Discussion by norbertjoni** [tteck/Proxmox#617] [Help Needed]
In the setup script or after the setup somewhere in the config can I disable ipv6 support?

**Comment by johntdyer** [comment]
yea, I am running into this same issue all of a sudden...

**Reply by johntdyer** [reply]
Figured out you can add an env var in the unit file and it will work

https://github.com/NginxProxyManager/nginx-proxy-manager/blob/546ce8d4bc0271b948b6a8532a4f71e6a2cb1dc7/docs/advanced-config/README.md#disabling-ipv6
